systemvm/patches/debian/config/opt/cloud/bin/merge.py

```diff
@@ -272,5 +275,10 @@ def setPath(self, path):
     def __moveFile(self, origPath, path):
         if not os.path.exists(path):
             os.makedirs(path)
-        timestamp = str(int(round(time.time())))
-        os.rename(origPath, path + "/" + self.fileName + "." + timestamp)
+
+        zipped_file_name = path + "/" + self.fileName + "." + str(uuid.uuid4()) + ".gz"
```

Comment:
The filename becomes quite long in that case. The date is already in the meta data of the file, so one could use `ls -latr` to get it:

![screen shot 2016-02-20 at 20 29 02](https://cloud.githubusercontent.com/assets/1630096/13198719/aa0d1982-d810-11e5-88c5-d833d738a442.png)